monitoring/cluster-monitoring/managing-alerts-using-monitoring-ui.adoc

```diff
@@ -1,16 +1,20 @@
-[id="managing-alerts-using-alertmanager-ui"]
-= Managing alerts using Alertmanager UI
+[id="managing-alerts-using-monitoring-ui"]
+= Managing alerts using Monitoring UI
 include::modules/common-attributes.adoc[]
 :context: managing-alerts
 
 toc::[]
 
-{product-title} 4 provides a Web interface to the Alertmanager, which enables you to manage alerts. This section demonstrates how to use the Alertmanager UI.
+{product-title} 4 provides a Web interface to the Alertmanager, which enables you to manage alerts. This section demonstrates how to use the Monitoring UI.
 
-include::modules/monitoring-contents-of-alertmanager-ui.adoc[leveloffset=+1]
+include::modules/monitoring-contents-of-monitoring-ui.adoc[leveloffset=+1]
 include::modules/monitoring-getting-information-about-alerts-and-alerting-rules.adoc[leveloffset=+1]
 include::modules/monitoring-silencing-alerts.adoc[leveloffset=+1]
 include::modules/monitoring-getting-information-about-silences.adoc[leveloffset=+1]
 include::modules/monitoring-editing-silences.adoc[leveloffset=+1]
 include::modules/monitoring-expiring-silences.adoc[leveloffset=+1]
 
+.Next steps
+
+xref:../../monitoring/cluster-monitoring/prometheus-alertmanager-and-grafana.adoc#prometheus-alertmanager-and-grafana[Access the Prometheus, Alertmanager, and Grafana.]
```

Comment:
@rh-max If you change the anchor text to: #accessing-prometheus from: #prometheus-alertmanager-and-grafana the build will pass for this particular error.

There is a big long explanation about this, and am happy to go into it when we see each other :)

Reply:
@vikram-redhat No comment. :)